render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlEntitySerializerTest.kt

```diff
@@ -0,0 +1,99 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.entity.Entities
+import com.linecorp.kotlinjdsl.querymodel.jpql.entity.impl.JpqlEntity
+import com.linecorp.kotlinjdsl.render.TestRenderContext
+import com.linecorp.kotlinjdsl.render.jpql.introspector.JpqlEntityDescription
+import com.linecorp.kotlinjdsl.render.jpql.introspector.JpqlRenderIntrospector
+import com.linecorp.kotlinjdsl.render.jpql.serializer.*
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+
+
```

Comment:
One unnecessary newline has been added.

Reply:
I wonder about something related to this. 
Do you have any plans to add linter like `ktlint`?

Reply:
I do intend to use klint, and I intend to use it just before or after the 3.0.0 release. This is because there are no samples for organizing code conventions. I thought it was too early to start thinking about what conventions would be good for Kotlin JDSL at this stage.